src/ContainsDuplicatesTest.java

```diff
@@ -0,0 +1,13 @@
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Consider writing another test for this that takes an array with no duplicates, just to be sure your function works for that, too :)